test/import-test.js

```diff
@@ -1030,6 +1031,43 @@ describe('#importConfigFromFilePath', () => {
   });
 });
 
+describe('#importCimcoreFromFilePath', () => {
+  it('should be able to correctly import specifications instance and then export to identical cimcore', () => {
+    const [importedConfigSpecifications, importedSpecifications] = importCimcoreFolder();
+
+    //This is the cimcore produced from importedSpecs. Used for verifying fidelity
+    const cimcoreSpecifications = convertSpecsToCimcore(importedConfigSpecifications, importedSpecifications);
+
+    const origProjectJSON = importCimcoreProjectFile();
+    expect(JSON.stringify(cimcoreSpecifications.projectInfo, null, 2)).to.eql(origProjectJSON);
+
+    //meta namespace files
+    for (const ns in cimcoreSpecifications.namespaces) { //namespace files
+      const origNsJSON = importCimcoreNSFile(ns);
+      expect(JSON.stringify(cimcoreSpecifications.namespaces[ns], null, 2)).to.eql(origNsJSON);
```

Comment:
So... this is doing string comparison of the JSON contents... It seems to work fine, but I think it might be better if we did object comparisons instead?  The good thing about object comparisons is that the order of the keys in the object (as well as whitespace) don't affect the results.

Anyway... something to think about.  We don't *have* to do it right now though.

Reply:
In theory, the order of keys should be identical as well since the generation process is identical, although I agree its not the most effective of comparisons. If I remember my thought process from a few weeks ago correctly, I did the string comparison because in practice javascript object comparison isn't entirely trivial. I'll put a comment in for now to let us know to consider it for the future.